raiden/network/stunsock.py

```diff
@@ -17,7 +17,7 @@ def stun_socket(
 ):
     timeout = socket.gettimeout()
     socket.settimeout(2)
-    log.debug('Initiating STUN for %s:%s', source_ip, source_port)
+    log.debug('Initiating STUN', source_ip=source_ip, source_port=source_port)
```

Comment:
`structlog` doesn't accept *args in `printf()` style, you have to use `%` to concat the message or provide `kwargs` like you did here